### PR TITLE
Fix docstring of NDCubeSequencePlotMixin.plot_as_cube

### PR DESCRIPTION
**Pull request by kakirastern**
### Description

To fix the formatting of the doctring in `NDCubeSequencePlotMixin.plot_as_cube` in `ndcube/mixins/sequence_plotting.py` to get rid of unexpected unindent and missing blank line after block quote errors.

### CODE REVIEW

**Comment by kakirastern** [comment]
I am seeing `E   sqlite3.OperationalError: database is locked` as the error/failure for the `py37 [macos]` tests.

**Comment by kakirastern** [comment]
And should I add a changelog for this PR?

**Comment by DanRyanIrish** [comment]
@kakirastern: Yep, add a "trivial" type changelog.

**Comment by DanRyanIrish** [comment]
> I am seeing `E sqlite3.OperationalError: database is locked` as the error/failure for the `py37 [macos]` tests.

I think this erro should be fixed by deleting the [```sunpy.map import```](https://github.com/kakirastern/ndcube/blob/fix-ndcubesequnce-plot-as-cube-doctring/ndcube/tests/test_ndcubesequence.py#L9).  It isn't used and shouldn't be there anyway.

**Comment by kakirastern** [comment]
> > I am seeing `E sqlite3.OperationalError: database is locked` as the error/failure for the `py37 [macos]` tests.
> 
> I think this erro should be fixed by deleting the [`sunpy.map import`](https://github.com/kakirastern/ndcube/blob/fix-ndcubesequnce-plot-as-cube-doctring/ndcube/tests/test_ndcubesequence.py#L9). It isn't used and shouldn't be there anyway.

Cool, just removed that unused `import sunpy.map` statement as well.

**Comment by kakirastern** [comment]
Interesting, got the `isort` check passed, but encountered the following instead: 
```
codestyle run-test: commands[1] | pre-commit run --all-files
flake8...................................................................Passed
isort....................................................................Passed
Check python ast.........................................................Passed
Check for case conflicts.................................................Passed
Trim Trailing Whitespace.................................................Passed
Check Yaml...............................................................Passed
Debug Statements (Python)................................................Passed
Check for added large files..............................................Passed
Fix End of Files.........................................................Failed
- hook id: end-of-file-fixer
- exit code: 1
- files were modified by this hook

Fixing changelog/262.trivial.rst

Mixed line ending........................................................Passed
ERROR: InvocationError for command /home/vsts/work/1/s/.tox/codestyle/bin/pre-commit run --all-files (exited with code 1)
___________________________________ summary ____________________________________
ERROR:   codestyle: commands failed
```

File: changelog/262.trivial.rst [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Fix docstring of ``NDCubeSequencePlotMixin.plot_as_cube`` to to get rid of
```
Comment (DanRyanIrish) [inline changelog/262.trivial.rst:1]:
```suggestion
Fix  docstring formatting to help docs build.
```

File: changelog/262.trivial.rst [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+unexpected unindent and missing blank line after block quote errors in
```
Comment (DanRyanIrish) [inline changelog/262.trivial.rst:2]:
```suggestion
```

File: changelog/262.trivial.rst [hunk 3]
```diff
@@ -0,0 +1,3 @@
...
+``ndcube/mixins/sequence_plotting.py``.
```
Comment (DanRyanIrish) [inline changelog/262.trivial.rst:3]:
```suggestion
```

File: changelog/262.trivial.rst (hunk 1 above)
Comment (DanRyanIrish) [reply changelog/262.trivial.rst:1]: This is quite verbose for a changelog.  Suggest condensing it to the one-liner, above.

**Comment by DanRyanIrish** [comment]
Wow, this looks like you really had to go down a couple rabbit holes!  Great job chasing it up.  Because of some of the files you've changed, I don't feel fully qualified to approve it until @nabobalis or @Cadair give it a quick look over.

But in the meantime, could you squash all the commits into the first changleog one except the first two?  I may include this PR in a bugfix release and a cleaner git history will make that easier.  Let me know if you aren't familiar with squashing commits and we can walk through it.

**Comment by kakirastern** [comment]
Sure, have just rebased and squashed all commits except the first two.

File: .pre-commit-config.yaml [hunk 4]
```diff
@@ -25,8 +25,13 @@ repos:
...
         args: ['--count', '--select', 'E101,W191,W291,W292,W293,W391,E111,E112,E113,E303,E304,E306,E502,E722,E901,E902,F822,F823']
 
+#  - repo: https://github.com/asottile/seed-isort-config
+#    rev: v2.1.0
+#    hooks:
+#      - id: seed-isort-config
+
   - repo: https://github.com/timothycrosley/isort
-    rev: 4.3.21-2
+    rev: 4.3.21
```
Comment (nabobalis) [inline .pre-commit-config.yaml:34]: Why was this changed?

File: .pre-commit-config.yaml [hunk 5]
```diff
@@ -25,8 +25,13 @@ repos:
       - id: flake8
         args: ['--count', '--select', 'E101,W191,W291,W292,W293,W391,E111,E112,E113,E303,E304,E306,E502,E722,E901,E902,F822,F823']
 
+#  - repo: https://github.com/asottile/seed-isort-config
+#    rev: v2.1.0
+#    hooks:
+#      - id: seed-isort-config
+
```
Comment (nabobalis) [inline .pre-commit-config.yaml:32]:
```suggestion
```

File: setup.cfg [hunk 6]
```diff
@@ -51,7 +51,7 @@ not_skip = __init__.py
 sections = FUTURE, STDLIB, THIRDPARTY, ASTROPY, FIRSTPARTY, LOCALFOLDER
```
Comment (nabobalis) [inline setup.cfg:51]:
```suggestion
sections = FUTURE, STDLIB, THIRDPARTY, FIRSTPARTY, LOCALFOLDER
```

File: .pre-commit-config.yaml (hunk 4 above)
Comment (kakirastern) [reply .pre-commit-config.yaml]: Sorry @nabobalis I was testing something. Have just reverted the change now.

**Comment by kakirastern** [comment]
@nabobalis I have just made the config changes you have suggested. After which the CI checks are passing again (especially the one for the tricky `sunpy.ndcube (codestyle [linux])`), so it is a good sign.

**Comment by kakirastern** [comment]
Just rebased and squashed again so that only the first two commits remain.
